spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -50,9 +50,9 @@
         # v0005
         expect(verification_array[5]).to eq(Moab::StorageObjectValidator::NO_XML_FILES =>"Version: v0005 Missing all required metadata files")
       end
-      it 'has missing content directory' do
+      it 'has missing metadata directory' do
```

Comment:
I like that this is tested here, but do we also have tests for when content directory is missing, and when it is present (both should pass, yes?)

Reply:
+1  for test with missing content dir